app/app.go

```diff
@@ -236,6 +241,9 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 	// init params keeper and subspaces
 	app.ParamsKeeper = initParamsKeeper(appCodec, cdc, keys[paramstypes.StoreKey], tkeys[paramstypes.TStoreKey])
 
+	// set the BaseApp's parameter store
+	bApp.SetParamStore(app.ParamsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramskeeper.ConsensusParamsKeyTable()))
```

Comment:
Just a note after speaking with @aaronc - most of the non `x/data` related changes like in this line were required to get a node to successfully start. Since we didn't have integration tests setup, we weren't catching these issues in CI.